.circleci/config.yml

```diff
@@ -1464,6 +1473,7 @@ workflows:
       - node-mongoose
       - node-mysql
       - node-net
+      - node-next
```

Comment:
I think there are 4 places in circle config that this should show up?

Reply:
I can't find where it would be missing.